reload_app/app.py

```diff
@@ -177,15 +178,15 @@ def event(self, request):
             try:
                 type_expected(data[field])
             except ValueError:
-                client.captureException()
+                sentry_sdk.capture_exception()
                 return Response("bad request maybe check field type\n", status=400)
 
             type_received = type(data[field])
             if type_expected != type_received:
-                client.captureMessage(
+                sentry_sdk.capture_message(
                     "expected %s, received %s for field %s of event %s"
                     % (type_expected, type_received, field, data["event_name"]),
-                    level="warning",
+                    "warning",
```

Comment:
```suggestion
                    level="warning",
```

The original would also work, if you prefer [explicitly naming the argument](https://github.com/getsentry/sentry-python/blob/832263bedca595be1e31a519d4f49f477bd77760/sentry_sdk/api.py#L82).